test/server/reporters-test.js

```diff
@@ -336,4 +352,12 @@ describe('Reporters', function () {
     it('xunit', function (done) {
         testReporter('xunit', done);
     });
+
+    it('Should xunit reporter use plain text decorator', function () {
+        testUsingPlainTextDecorator('xunit');
+    });
+
+    it('Should json reporter use plain text decorator', function () {
```

Comment:
It's still incorrect, what about
```js
it('Should force plain text decorator for reporters that target machine-readable formats')
```
which read as
```
It should force plain text decorator for reporters that target machine-readable formats
```
and it makes sense, opposite to:
```
It should reporters that target machine-readable formats always use the plain text decorator
```